app/views/books/_books_search.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= form_tag(books_path, method: :get, id: 'search_form') do %>
+  <%= text_field_tag :search, params[:search], placeholder: "Search by book or by author" %>
+  <%= submit_tag "Search", name: nil %>
```

Comment:
Can we omit the `name` option entirely here?

Reply:
@mxie, I added `name: nil` because it makes for a cleaner url. If you don't have `name: nil` the url has a search param as follows `search=john?"commit"="search"`

Reply:
Ohh I see. It's because you're doing a get request...I think once you extract the searches controller, this problem will go away.